FinSearchUnified/Tests/PluginTest.php

```diff
@@ -90,13 +89,52 @@ public function testArticleExport($articlesActiveStatus, $expectedCount, $errorM
         $this->assertEquals($expectedCount, $actual, sprintf($errorMessage, $actual));
     }
 
+    public function crossSellingCategoryProvider()
+    {
+        return [
+            'No cross-sell categories configured' => [
+                'crossSellingCategories' => [],
+                'expectedCount' => 1
+            ],
+            'Article does not exist in cross-sell category configured' => [
+                'crossSellingCategories' => [5],
+                'expectedCount' => 1
+            ],
+            'Article exists in one of the cross-sell categories configured' => [
+                'crossSellingCategories' => [8, 9],
+                'expectedCount' => 1
+            ],
+            'Article exists in all of cross-sell categories configured' => [
```

Comment:
I think it was a little but confusing, but actually the count should be `0` for all cases, except for the first one. As long as one cross sell category matches the product, it should not be exported.
The product has the categories 8, 9 and 10, which means if any of these categories are declared as a cross-sell category, the product should not be exported.

For example:
This means if the category 5 is assigned, which may be a child of 1 and the configured cross-sell category is 1, then the product should still be exported. If on the other hand, 5 is set as cross-sell category, the product should not be exported.

Reply:
@TheKeymaster https://findologic.myjetbrains.com/youtrack/issue/SW-377#focus=streamItem-94-24651.0-0
This would require re-implementing the story as I dont think it would be possible through query alone. We will have to use the `shouldBeExported` flag for this which makes the current implementation useless.